user/api.go

```diff
@@ -376,12 +417,14 @@ func (a *Api) UpdateUser(res http.ResponseWriter, req *http.Request, vars map[st
 				}
 			}
 
-			if updatedUser.EmailVerified && updatedUser.TermsAccepted != "" {
+			if updatedUser.EmailVerified && updatedUser.TermsAccepted != "" && !strings.HasSuffix(updatedUser.Username, "@tidepool.io") && !strings.HasSuffix(updatedUser.Username, "@tidepool.org") {
```

Comment:
I think you should publish events about all users, even if the email is not verified, terms are not accepted or the email is a tidepool email address. It's should be the responsibility of the marketo consumer to filter out events which do not match those conditions.

Reply:
@toddkazakov 2 main reasons I have this and why here. 

1. The email verified and terms accepted are to check if a user is fully created and verified before we put them in the email campaigns for marketo. Did not think it made sense to send a kafka message for marketo to just not read.

2. The only reason I added the tidepool email address check here (it is already in marketo) is because ginny is creating and deleting test accounts around every 3 minutes or so and for the same reason as above thought it was unnecessary logging, messages, and noise.

Reply:
can also put all of this into its own `marketo` function to call here

Reply:
Yep, makes sense. However, I'd like this topic to be used for all user updates, not only for marketo.

Reply:
just a comment test for tidebot